python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -339,7 +339,10 @@ def default_asset_key_fn(dbt_resource_props: Mapping[str, Any]) -> AssetKey:
         dbt models: a dbt model's key is the union of its model name and any schema configured on
             the model itself.
     """
-    dagster_metadata = dbt_resource_props.get("meta", {}).get("dagster", {})
+    dbt_meta: Any | Mapping[str, Any] | dict[Any, Any] = dbt_resource_props.get("config", {}).get(
```

Comment:
I feel like this type annotation doesn't actually do much on account of one of the options being `Any`

Reply:
oops this was added by the editor